.gitignore

```diff
@@ -56,4 +56,5 @@ typings/
 .yarn-integrity
 
 # dotenv environment variables file
-.env
\ No newline at end of file
+.env
+.env.example

```

Comment:
So the dockerfile actually copies and uses .env.example right, and the JS project uses .env.example? Normally example files don't actually contribute to the actual build/run. In that case, why not just name it .env and keep some helpful comments on the top of the file